packages/terra-clinical-label-value-view/src/LabelValueView.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import './LabelValueView.scss';
+
+const propTypes = {
+  /**
+   * The label of the LabelValueView.
+   */
+  label: PropTypes.string.isRequired,
+  /**
+  * The text to be displayed underneath the label with the provided styling.
+  */
+  textValue: PropTypes.string,
+  /**
+   *  Child component(s) to display underneath the label.
+   */
+  children: PropTypes.element,
```

Comment:
Yeah, I wasn't sure when I changed this. [Changed Back Here](https://github.com/cerner/terra-clinical/pull/3/commits/be4eaacb32bf32793566aebbf851eef3179064ea#diff-00cdf59bc370c025c9c00e3dbd5c61d0R41)